### DISCUSSION TITLE
Needs option not to close the picker after creating.

### DISCUSSION THREAD

**Discussion by HubKing** [uBlockOrigin/uBlock-issues#3057] [Ideas] [answered]
Often, a web page contains more than 1 elements I do not want. Currently, it seems that the picker gets closed once I click "Create", and I have to restart it from the beginning. This is already cumbersome, but on FireFox Android, this is even more cumbersome. 

How about adding an option not to close the picker after creating? The user can easily close it by clicking "Quit" anyway.

**Comment by u-RraaLL** [comment, marked as answer]
Duplicate https://github.com/uBlockOrigin/uBlock-issues/discussions/1929

As a workaround, open the logger and open uBO popup in it. Switching between the website and the logger to relaunch picker is less work.

**Reply by garry-ut99** [reply]
> Duplicate https://github.com/uBlockOrigin/uBlock-issues/discussions/1929

Rather a duplicate of : 
- https://github.com/uBlockOrigin/uBlock-issues/issues/467 

(and https://github.com/uBlockOrigin/uBlock-issues/discussions/1929 can be considered a duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/467 as well)
> As a workaround (...)

Yes, but if neither of both (hot-key and logger workaround) are easy enought for some users, then an alternative and the easiest & least cumbersome solution, would be to add an additional 2in1 `Create&Pick` button:

<img src="https://github.com/uBlockOrigin/uBlock-issues/assets/72945564/2acf8726-6f33-45ba-b0ef-43888a225b65" width="500">

**Reply by u-RraaLL** [reply]
> Rather a duplicate of (...)

Not exactly. The issue was for desktop and the original title was looking for a shortcut (while keeping picker open was just an alternative), Enabling a shortcut was explained and the issue was closed as fixed.

> are easy enought for some users

True, which is why I kept the discussion open.

> then an alternative and the easiest & least cumbersome solution, would be to add an additional 2in1 `Create&Pick` button

On the contrary, this would be totally cumbersome.

The main reason to keep picker open would be to improve the ease of use on mobile. There is **no space** for more buttons on mobile

Anyway, let's continue in the discussion I linked..

**Comment by gorhill** [comment]
You can assign a keyboard shortcut to launch the element picker: <https://github.com/gorhill/uBlock/wiki/Keyboard-shortcuts>

**Reply by u-RraaLL** [reply]
Not for Android, which he also mentions.
